_episodes_rmd/09-vector-when-data-dont-line-up-crs.Rmd

```diff
@@ -1,8 +1,13 @@
 ---
-layout: post
-title: "Vector 03: When Vector Data Don't Line Up - Handling Spatial
-Projection & CRS in R"
-date:   2015-10-24
+title: "When Vector Data Don't Line Up - Handling Spatial Projection & CRS in R"
+teaching: 10
+exercises: 0
+questions:
+- ""
+objectives:
```

Comment:
we used to have objective at the top of each page in the grey box. Is this just a yaml list? should be easy to populate even if we use the online neon lessons...